### PR TITLE
Marauder

### PR DESCRIPTION
**Pull request by BB-66**
New hand-wired project - build log [here](https://imgur.com/a/TvsXpq5).

## Description

90% hand-wired mechanical keyboard in an old Razer case; built around an Elite-C v3.1 controller with VIA support.

## Types of Changes

- [x] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/marauder/config.h [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4D52
+#define PRODUCT_ID      0x0000
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    %MANUFACTURER%
```
[the remark this answers is not in the document]
Comment (BB-66) [reply keyboards/handwired/marauder/config.h]: Changed PRODUCT_ID to 0x0001 and %MANUFACTURER% to %BB-66% (I'm unsure if it's correct or needs to be BB-66 alone).

I grepped 0x4D52 and there was no return in the QMK directory so I used this as VENDOR_ID (I understand this needs to be unique).

Please excuse my inexperience with git and thank you for the help!

File: keyboards/handwired/marauder/keymaps/via/marauder.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
```
[the remark this answers is not in the document]
Comment (BB-66) [reply keyboards/handwired/marauder/keymaps/via/marauder.json]: I removed the VIA directory entirely and will explore how to add the VIA support after the basic gets added to QMK. Thank you!

[the remark this answers is not in the document]
Comment (drashna) [reply keyboards/handwired/marauder/keymaps/via/marauder.json]: The via folder is fine, it's the json that is the problem.   The via folder should contain the keymap.c file, a rules.mk file with just `VIA_ENABLE = yes`, and whatever other options enabled, and optionally, a config.h.   Nothing else.

File: keyboards/handwired/marauder/config.h [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4D52
+#define PRODUCT_ID      0x0001
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    %BB-66%
```
Comment (drashna) [inline keyboards/handwired/marauder/config.h:25]: Should just be: 
```suggestion
#define MANUFACTURER    BB-66
```

File: keyboards/handwired/marauder/info.json [hunk 4]
```diff
@@ -0,0 +1,108 @@
+{
+    "keyboard_name": "Minshara Marauder",
+    "url": "",
+    "maintainer": "qmk",
```
Comment (drashna) [inline keyboards/handwired/marauder/info.json:4]: These should be filled out too.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/marauder/keymaps/via/marauder.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (BB-66) [reply keyboards/handwired/marauder/keymaps/via/marauder.json]: Will apply this after the initial pull is sorted out. At the moment I'm attempting to add an OLED screen and sort out the RGB, will implement new features as time goes by. Thank you so much!

File: keyboards/handwired/marauder/info.json (hunk 4 above)
Comment (BB-66) [reply keyboards/handwired/marauder/info.json]: Sorted with a link to the build project and filling in my name under maintainer, hope this is all right. Thank you for your patience!

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**